File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.java
class Solution {
    public int solution(int[] dot) {
        int answer = 0;
        int one=0;//x좌표
        int two=0;//y좌표
        if(dot[0]>answer){
            one++;
        }
        if(dot[1]>answer){
            two++;
        }
        if(one==1 && two==1){
            //1사분면
            answer=1;
        }
        else if(one==0 && two==1){
            //2사분면
            answer=2;
        }
        else if(one==0 && two==0){
            //3사분면
            answer=3;
        }
        else if(one==1 && two==0){
            //4
            answer=4;
        }
        return answer;
    }
}

File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.py
def solution(dot):
    answer = 0
    x=0
    y=0
    if dot[0]>0 : #x가 양수일때
        x=1
    if dot[1]>0 : #y가 양수일때
        y=1
    if (x==1 and y==1) : #(+ , +) 1사분면일 때
        answer=1
    elif (x==0 and y==1 ) :
        answer=2
    elif (x==0 and y==0) :
        answer=3
    elif (x==1 and y==0) :
        answer=4
    return answer